docs/tutorial_tips_tricks.rst

```diff
@@ -161,6 +161,45 @@ If we run the test, we can see that the correct standard output filename will be
    [==========] Finished on Wed Jan 20 17:19:01 2021
 
 
+Debugging sanity and performance patterns
+=========================================
+When creating a new test that requires a complex output parsing for either the sanity or performance stages, setting the :attr:`sanity_patterns` and :attr:`perf_patterns` may involve some trial and error to debug the complex regular expressions required.
+For lightweight tests which execute in a few seconds, this trial and error may not be an issue at all.
+However, when dealing with tests which take longer to run, this method can quickly become tedious and inefficient.
+
+When a test fails, ReFrame will keep the test output in the stage directory after its execution, which means that one can load this output into a Python shell or another helper script without having to rerun the expensive test again.
+If the test is not failing but the user still wants to experiment or modify the existing :attr:`sanity_patterns` or :attr:`perf_patterns`, the command line option ``--keep-stage-files`` can be used when running ReFrame to avoid deleting the stage directory.
```

Comment:
```suggestion
If the test is not failing but the user still wants to experiment or modify the existing :attr:`sanity_patterns` or :attr:`perf_patterns`, the command line option :option:`--keep-stage-files` can be used when running ReFrame to avoid deleting the stage directory.
```

Reply:
And we could also make `sanity_patterns` and `perf_patterns` links.